Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:latest
+
+ENV config_dir=/root/.chiadog/config.yaml
+ENV TZ=UTC
+
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install -y python3 bash ca-certificates git openssl python3-pip build-essential python3-dev python3.8-venv python3.8-distutils python-is-python3 tzdata
+
+RUN dpkg-reconfigure -f noninteractive tzdata
+
+RUN apt-get install -y libsodium-dev
+RUN SODIUM_INSTALL=system pip3 install pynacl
```

Comment:
Why do we install `libsodium` separately from `pynacl`?

From [docs](https://pynacl.readthedocs.io/en/latest/install/) it seems like `pip install pynacl` should replace both lines above.

Reply:
This was imported as-is from ajacobson/chiadog-docker so @ajacobson might have more insight if there's some magic to it, but from what I can see it should not be required assuming we can install the binary wheel. Since the base is just `ubuntu:latest` I don't see why that wouldn't work. Will simplify & test.

Reply:
Actually we don't need to install pynacl separately at all since the requirements.txt will pull it in straight to the venv. So this gets simplified quite a bit.